packages/dioxus-translate-macro/src/lib.rs

```diff
@@ -244,6 +259,7 @@ pub fn translate_derive(input: TokenStream) -> TokenStream {
     // Generate the implementation block for `translate`
     let gen = quote! {
         impl #enum_name {
+            pub const VARIANTS: &'static [Self] = &[ #(#enum_name::#idents),* ];
```

Comment:
```suggestion
            pub const VARIANTS: &'static [Self] = &[ #(#enum_name::#idents),* ];
            pub fn variants(lang: &dioxus_translate::Language) -> Vec<String> {
                  Self::VARIANTS
                      .iter()
                      .map(|v| v.translate(lang).to_string())
                      .collect()
            }
```
보니깐 많이 사용하는 패턴 같아서, 이렇게 해둬도 좋지 않을까요?

Reply:
@hackartists 네 좋은 것 같습니다!